studio/src/pages/RequestorPage/ResponsePanel/ResponsePanel.tsx

```diff
@@ -182,7 +182,7 @@ export function ResponsePanel({
                         ) : (
                           <CaretRightIcon className="w-4 h-4 cursor-pointer" />
                         )}
-                        Timeline
+                        Logs & events
```

Comment:
```suggestion
                        Logs & Events
```

capitalizing looks nice to me